src/main/groovy/org/sharegov/mdcgis/PropertyInfoService.groovy

```diff
@@ -87,7 +84,19 @@ class PropertyInfoService {
 
 	}
 
+	/**
+	 * Get Property information by Folio.
+	 *
+	 * @param folio
+	 * @return gets folio or returns null.
+     */
+	Map getPropertyInfoByFolio(String folio){
 
+		Map propertyLayers = ['MDC.HomeOwnerAssociation':null,
+							  'MDC.PaParcel':null,
+							  'MDC.PaGIS':getCleanPropertyInfoByFolio(folio)]
```

Comment:
There was a misunderstanding - corrected as discussed. Now it will return as suggested.